spec/models/service_template_spec.rb

```diff
@@ -845,21 +845,19 @@
 
     context "#provision_request" do
       let(:arg1) { {'ordered_by' => 'fred'} }
-      let(:arg2) { {:initiator => 'control'} }
+      let(:arg2) { {:initiator => 'control', :submit_workflow => true} }
 
-      it "provision's a service template without errors" do
+      it "provisions a service template without errors" do
         expect(resource_action_workflow).to receive(:validate_dialog).and_return([])
         expect(resource_action_workflow).to receive(:make_request).and_return(miq_request)
-        expect(resource_action_workflow).to receive(:set_value).with('ordered_by', 'fred')
-        expect(resource_action_workflow).to receive(:request_options=).with(:initiator => 'control')
+        expect(resource_action_workflow).to receive(:request_options=).with(:initiator => 'control', :submit_workflow => true)
 
         expect(service_template.provision_request(user, arg1, arg2)).to eq(miq_request)
       end
 
-      it "provision's a service template with errors" do
+      it "provisions a service template with errors" do
         expect(resource_action_workflow).to receive(:validate_dialog).and_return(%w(Error1 Error2))
-        expect(resource_action_workflow).to receive(:set_value).with('ordered_by', 'fred')
-        expect(resource_action_workflow).to receive(:request_options=).with(:initiator => 'control')
+        expect(resource_action_workflow).to receive(:request_options=).with(:initiator => 'control', :submit_workflow => true)
```

Comment:
@d-m-u - It would be great to see a spec where `:submit_workflow` is not passed in and what behavior we see there.  I'm still a bit concerned that this extra option could create a regression somewhere we're not expecting.